messages/project.deploy.functions.md

```diff
@@ -0,0 +1,9 @@
+# summary
+  
+Deploy all functions in this project to a specified compute environment.
```

Comment:
Starter content for this command can be found here: https://salesforce.quip.com/2gB5ASLb3uaD#SaFACADNw49